<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ipw::ImagePlot</class>
 <widget class="QWidget" name="ipw::ImagePlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QwtScaleWidget" name="scaleLeft" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QwtScaleWidget" name="scaleBottom" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsView" name="view">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtScaleWidget</class>
   <extends>QWidget</extends>
   <header>qwt/qwt_scale_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
